Sources/SwiftGRPCNIO/GRPCServerCodec.swift

```diff
@@ -0,0 +1,66 @@
+import Foundation
+import SwiftProtobuf
+import NIO
+import NIOFoundationCompat
+import NIOHTTP1
+
+public enum GRPCServerRequestPart<MessageType: Message> {
```

Comment:
Would be awesome if we could add docstrings for these

Reply:
Done.

Reply:
Done.